modules/docs/lib/widgetUtils.tsx

```diff
@@ -102,31 +157,69 @@ export const SymbolDialog = ({value}: SymbolDialogProps) => {
         >
           {value.displayName || value.name}
         </Dialog.Target>
+
         {renderTypeParameters(value.typeParameters)}
-      </>
-      <Dialog.Popper>
-        <Dialog.Card maxHeight="50vh" maxWidth="90vh">
-          <Dialog.CloseIcon />
-          <Dialog.Heading>{value.name}</Dialog.Heading>
-          <Dialog.Body>
-            <RenderContext.Provider value="table">
-              <IndentLevelContext.Provider value={0}>
-                {symbol ? (
-                  <SymbolDoc name={value.name} headingStart={3} hideHeading />
-                ) : (
-                  <>
-                    <p>Basic type information:</p>
-                    <pre>
-                      <code>{value.value}</code>
-                    </pre>
-                  </>
-                )}
-              </IndentLevelContext.Provider>
-            </RenderContext.Provider>
-          </Dialog.Body>
-        </Dialog.Card>
-      </Dialog.Popper>
-    </Dialog>
+
+        <Dialog.Popper>
+          <Dialog.Card maxHeight="50vh" maxWidth="90vh">
+            <Dialog.CloseIcon />
+
+            <Dialog.Heading>
+              {value.name}{' '}
+              {breadcrumbsList.length >= 1 ? (
+                <Breadcrumbs aria-label="Breadcrumbs">
+                  <Breadcrumbs.List>
+                    {breadcrumbsList.map((item, index) => {
+                      return (
+                        <>
+                          {item === breadcrumbsList[breadcrumbsList.length - 1] ? (
+                            <Breadcrumbs.CurrentItem key={index}>{item}</Breadcrumbs.CurrentItem>
+                          ) : (
+                            <Breadcrumbs.Item key={index}>
```

Comment:
Without knowing this is a breadcrumb with a single item, the smaller `SecondaryButton` might seem like a rendering error where the heading is being duplicated:

<img width="713" alt="Screenshot 2023-03-08 at 6 41 11 PM" src="https://user-images.githubusercontent.com/146020/223902238-17701e41-acbb-46fd-8caf-53d713953b9c.png">

Would it be wrong to only show the breadcrumbs if there's more than one item?

Reply:
i think that makes sense!